src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -515,7 +515,7 @@ public function testDeleteOfParent()
             '/api/categories'
         );
 
-        //$this->assertEquals(200, $client->getResponse()->getStatusCode());
+        //$this->assertHttpStatusCode(200, $client->getResponse());
```

Comment:
can you remove this line? i know it was also commented out before but i dont like outcommented code (: